code/src/css/PopularList.css

```diff
@@ -0,0 +1,51 @@
+.list{
+    display:flex;
+    flex-wrap: wrap;
+    width: 100vw;
+  }
+  
+  .poster {
+    width: 25vw;
+  }
+  
+  .movieCard {
+    position: relative;
+  }
+  
+  .details {
+    position: absolute;
+    bottom: 1%;
+    color: #fff;
+    padding: 5%;
+  }
+
+  h1 {
+    font-size: 32px;
+  }
+
+  * {box-sizing: border-box}
+
+/* Container needed to position the overlay. Adjust the width as needed */
+.imageContainer {
+  position: relative;
+}
+
+/* The overlay effect - lays on top of the container and over the image */
+.imageOverlay {
+  position: absolute;
+  top: 0%;
+  left: 0%;
+  background: rgb(0, 0, 0);
+  background: rgba(0, 0, 0, 0.5); /* Black see-through */
+  color: #f1f1f1;
+  width: 100%;
+  height: 100%;
+  transition: .5s ease;
+  opacity:0;
+  color: white;
+}
+
+/* When you mouse over the container, fade in the overlay title */
+.imageContainer:hover .imageOverlay {
+  opacity: 1;
```

Comment:
Nicely done with the overlay. It's easy to understand and made the list look very nice. We just think it's a bit hard to read so maybe the overlay could be a little bit darker with higher opacity.

Reply:
Good idea, thanks!